Protocol.md

```diff
@@ -273,6 +273,8 @@ rules MAY be used for handling layout. For example, `application/x.netflix+cbor`
 * __Setup Data__: includes payload describing connection capabilities of the endpoint sending the
 Setup header.
 
+__NOTE__: A server that receives a SETUP frame that has (__R__)esume Enabled set, but does not support resuming operation, must reject the SETUP with an ERROR. 
```

Comment:
How is returning RESUME_NOT_SUPPORTED different than returning CONNECTION_ERROR_NO_RETRY? In either case the ReactiveSocket either gives up or reconnects with a SETUP frame.